src/__internal__/checkable-input/checkable-input.component.tsx

```diff
@@ -62,12 +64,15 @@ export interface CheckableInputProps extends CommonCheckableInputProps {
   labelAlign?: "left" | "right";
   /** When true label is inline */
   labelInline?: boolean;
+  /** [Legacy] Flag to configure component as optional in Form */
```

Comment:
comment: you don't need to define this twice, the common interface is extended so the type exists on the `CheckableInputProps` interface as a result

Reply:
This still needs doing, when I pass `isOptional` to Checkbox the TS compiler complains etc. Also noticed `InlineInputs` doesn't support it but probably should